Marlin/src/HAL/HC32/HAL.h

```diff
@@ -148,6 +148,18 @@
 #define GET_PIN_MAP_INDEX(pin) pin
 #define PARSED_PIN_INDEX(code, dval) parser.intval(code, dval)
 
+//
+// Debug port disable
+// JTMS / SWDIO = PA13
+// JTCK / SWCLK = PA14
+// JTDI         = PA15
+// JTDO         = PB3
+// NJTRST       = PB4
+//
+#define JTAGSWD_RESET() PORT_DebugPortSetting(0x1F, Enable);    // enable JTDO, JTDI, NJTRST, SWDIO, SWCLK
+#define JTAG_DISABLE() PORT_DebugPortSetting(0x1C, Disable);    // disable JTDO, JTDI, NJTRST
+#define JTAGSWD_DISABLE() PORT_DebugPortSetting(0x1F, Disable); // disable JTDO, JTDI, NJTRST, SWDIO, SWCLK
```

Comment:
What do you think of using the named variables here for the pins?

```suggestion
#define JTAGSWD_RESET() PORT_DebugPortSetting(ALL_DBG_PIN, Enable);
#define JTAG_DISABLE() PORT_DebugPortSetting(TDO_SWO | TDI | TRST, Disable);
#define JTAGSWD_DISABLE() PORT_DebugPortSetting(ALL_DBG_PIN, Disable);
```

Reply:
looks a lot more readable than using "magic" values. I'll update the PR accordingly

Reply:
Sorry for the delay, I had seen your comment but didn't realize the change has been made.

Reply:
The bit names I used in my recommendation were actually from the HC32 headers, I don't think you had to redefine them.

That said, your bit names are a little more consistent than those, so I think it is fine to go ahead and use what you've done in your most recent change.